src/pages-and-resources/discussions/app-config-form/apps/shared/DivisionByGroupFields.jsx

```diff
@@ -53,14 +53,14 @@ function DivisionByGroupFields({
                 onChange={onChange}
                 onBlur={onBlur}
                 checked={values.divideGeneralTopic}
-                label="General"
+                label={intl.formatMessage(messages.divideGeneralTopic)}
               />
               <Form.Check
                 id="divideQuestionsForTAs"
                 onChange={onChange}
                 onBlur={onBlur}
                 checked={values.divideQuestionsForTAs}
-                label="Questions for the TAs"
+                label={intl.formatMessage(messages.divideQuestionsForTAsTopic)}
```

Comment:
does it matter that this is a different attribute than the `values.divideQuestionsForTAs` at L62?

Reply:
Ugh, going to fix that.  They don't _need_ to be the same, but right now they're inconsistent with how the "divideGeneralTopic" one is.  Good eyeballs.